recon_surf/recon-surf.sh

```diff
@@ -641,7 +632,7 @@ if [ "$get_t1" == "1" ]
 then
   # create T1.mgz from nu (!! here we could also try passing aseg?)
   cmd="mri_normalize -g 1 -seed 1234 -mprage $mdir/nu.mgz $mdir/T1.mgz"
-  if [ "$vox_size" == "auto" ]
+  if [ "$vox_size" -lt "$hires_voxsize_threshold" ]
```

Comment:
Didn't we want to get rid of this and just return True/False for voxsize instead of the actual voxel size? And have the check with the hires_voxsize_threshold in conform.py

Reply:
I think it is better to do this here, so you can change the threshold in recon-surf